lib/Elastica/QueryBuilder.php

```diff
@@ -33,6 +33,7 @@ public function __construct(Version $version = null)
         $this->_version = $version ?: new Version\Latest();
 
         $this->addDSL(new DSL\Query());
+        // FIXME XXX: make it lazy
```

Comment:
If we don't make it "lazy" it means every time the query build is called, we get the deprecated error, right?